src/main/java/bisq/core/payment/validation/altcoins/bch/BchAddressValidator.java

```diff
@@ -0,0 +1,35 @@
+package bisq.core.payment.validation.altcoins.bch;
+
+import bisq.core.payment.validation.AbstractSpecificAltCoinAddressValidator;
+import bisq.core.payment.validation.params.btc.BtcMainNetParamsForValidation;
+import bisq.core.util.validation.InputValidator;
+import org.bitcoinj.core.Address;
+import org.bitcoinj.core.AddressFormatException;
+
+public class BchAddressValidator extends AbstractSpecificAltCoinAddressValidator {
```

Comment:
I'm not sure if we should use currency name (BitcoinCashAddressValidator) or a symbol, and if symbol then all letters in upper case or like it's now?

Reply:
Use the ticker symbol, but do standard capitalization, not all caps.